test/wrapper_array.jl

```diff
@@ -150,9 +150,10 @@ end
 
 const cnda = NamedDimsArray([10 20; 30 40], (:x, :y))
 @testset "allocations: wrapper" begin
-    @test 0 == @allocated parent(cnda)
-    @test 0 == @allocated dimnames(cnda)
+    @test 0 == @ballocated parent(cnda)
+    @test 0 == @ballocated dimnames(cnda)
 
+    # These tests use `@allocated` as for some reason `@ballocated` reports 1 alloc
     @test 0 == @allocated refine_names(cnda, (:x, :y))
```

Comment:
This is a bit odd.
I think it relates to https://github.com/JuliaLang/julia/issues/34871

Reply:
https://github.com/JuliaLang/julia/issues/14955